### PR TITLE
nixos-23.11: unbreak hopenpgp-tools

### PR DESCRIPTION
**Pull request by motiejus**
Unbreak hopenpgp-tools to 23.11

Master PR: https://github.com/NixOS/nixpkgs/pull/275414

### CODE REVIEW

**Review by dotlambda — DISMISSED** [review]
Plesse read https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#how-to-backport-pull-requests

File: pkgs/development/haskell-modules/hackage-packages.nix [hunk 1]
```diff
@@ -152448,7 +152448,6 @@ self: {
        description = "hOpenPGP-based command-line tools";
        license = lib.licenses.agpl3Plus;
        hydraPlatforms = lib.platforms.none;
```
Comment (cdepillabout) [inline pkgs/development/haskell-modules/hackage-packages.nix:152450]: You'll need to remove this line as well, otherwise hydra won't build the package.

**Comment by motiejus** [comment]
> Plesse read https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#how-to-backport-pull-requests

Thanks for the link and sorry for the noise. I've updated the PR following the guide to the best of my ability.

File: pkgs/development/haskell-modules/hackage-packages.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/development/haskell-modules/hackage-packages.nix:152450]: Shouldn't that be done on master first?

Comment (motiejus) [reply pkgs/development/haskell-modules/hackage-packages.nix:152450]: This line is not present on master, as the file was regenerated before the merge from haskell-updates.

**Comment by cdepillabout** [comment]
@ofborg build haskellPackages.hopengpg-tools

**Comment by cdepillabout** [comment]
@ofborg build haskellPackages.hopenpgp-tools

**Review by cdepillabout — APPROVED** [review]
Looks like ofborg built this cleanly.

Thanks for backporting this!

**Pull request merged**